google/cloud/bigquery/v2/minimal/internal/json_utils.h

```diff
@@ -51,6 +51,28 @@ void SafeGetTo(ResponseType& value, nlohmann::json const& j,
   auto i = j.find(key);
   if (i != j.end()) i->get_to(value);
 }
+
+template <typename T>
+std::shared_ptr<T> SafeGetTo(std::shared_ptr<T>& value, nlohmann::json const& j,
+                             std::string const& key) {
+  auto i = j.find(key);
+  if (i == j.end()) return value;
+  if (value == nullptr) {
+    value = std::make_shared<T>();
+  }
+  i->get_to(*value);
+  return value;
+}
+
+template <typename T>
+bool SafeValueOr(T& value, nlohmann::json const& j, std::string const& key) {
```

Comment:
We don't need `void SafeGetTo(...)` and `bool SafeValueOr(...)`. We should just modify `SafeGetTo` to return a `bool` like this function does.

https://github.com/googleapis/google-cloud-cpp/blob/916b808c9e918807dd3abd09049dc72e2e1bfd0c/google/cloud/bigquery/v2/minimal/internal/json_utils.h#L48-L53